javaagent/build.gradle.kts

```diff
@@ -45,7 +48,8 @@ val licenseReportDependencies by configurations.creating {
 
 dependencies {
   bootstrapLibs(project(":instrumentation-api"))
-  // opentelemetry-api is an api dependency of :instrumentation-api, but opentelemetry-api-logs is not
+  // opentelemetry-api is an api dependency of :instrumentation-api, but opentelemetry-api-(events|logs) is not
+  bootstrapLibs("io.opentelemetry:opentelemetry-api-events")
```

Comment:
is events needed, or just pulled in for completeness (no prob)?

Reply:
Not needed, I just noticed that the event API was being shaded and kept in the agent class loader, figured we should probably move it to bootstrap as any other API dep.